pulser/tests/test_simulation.py

```diff
@@ -19,10 +19,10 @@
 
 import qutip
 
-from pulser import Sequence, Pulse, Register
+from pulser import Sequence, Pulse, Register, Simulation
```

Comment:
You should keep the import of `Simulation` from `pulser.simulation`, it's the default way. We might deprecate the `from pulser` import at some point.